### DISCUSSION TITLE
How to change the configuration files from `.oh-my-bash` to `.config/oh-my-bash`?

### DISCUSSION THREAD

**Discussion by moshpirit** [ohmybash/oh-my-bash#636] [Q&A]
I don't like having a bunch of config files at the home directory, I prefer to keep my config files at `~/.config`. How can I do that? Since OMB notifies of a new version and downloads it too in this directory, I would like to keep having this feature, but in `~/.config/oh-my-bash`.

**Comment by akinomyoga** [comment]
It is described in [README](https://github.com/ohmybash/oh-my-bash?tab=readme-ov-file#custom-directory).

**Reply by moshpirit** [reply]
Sorry, didn't see that. Thanks for replying!
